package.json

```diff
@@ -11,9 +11,12 @@
     "url": "https://wordpress.org/support/plugin/facebook-for-woocommerce"
   },
   "devDependencies": {
+    "@wordpress/scripts": "^14.0.0",
     "sake": "github:skyverge/sake"
   },
   "scripts": {
-    "generate:category_attribute_json": "php bin/GenerateCategoryAttributeMapping.php"
+    "generate:category_attribute_json": "php bin/GenerateCategoryAttributeMapping.php",
+    "build": "NODE_ENV=production wp-scripts build",
```

Comment:
We should probably reserve `npm run build` for full build for compatibility with WooRelease - i.e. full clean build, with all dependencies (npm, composer etc). Could use `npm run build:client` (or similar) for the client build specifically.

Reply:
Good call. Renamed to `build:assets` (same as Woo core) in 6b5cf87786e8ececce1747056b6a6cbfe1684c6b.